java/memory/memory-core/src/main/java/org/apache/arrow/memory/util/MemoryUtil.java

```diff
@@ -142,7 +142,7 @@ public Object run() {
       // the static fields above get initialized
       final RuntimeException failure = new RuntimeException(
           "Failed to initialize MemoryUtil. You must start Java with " +
-              "`--add-opens=java.base/java.nio=ALL-UNNAMED` " +
+              "`--add-opens=java.base/java.nio=org.apache.arrow.memory.core,ALL-UNNAMED` " +
```

Comment:
would this require a documentation update?

Reply:
should we include it to this PR or file a separate issue for that?

Reply:
I think it's best to do a separate PR for doc updates around this since there are a few of these across multiple PRs.